adal/build/LayoutNugetPackage.proj

```diff
@@ -46,140 +45,143 @@
+    <SourceFile Include="$(MSBuildThisFileDirectory)..\src\Microsoft.IdentityModel.Clients.ActiveDirectory.Platform\bin\$(Configuration)\net45\Microsoft.IdentityModel.Clients.ActiveDirectory.Platform.dll">
       <TargetFolder>net45</TargetFolder>
     </SourceFile>
-    <SourceFile Include="..\src\Microsoft.IdentityModel.Clients.ActiveDirectory.Platform\bin\$(Configuration)\net45\Microsoft.IdentityModel.Clients.ActiveDirectory.Platform.pdb">
+    <SourceFile Include="$(MSBuildThisFileDirectory)..\src\Microsoft.IdentityModel.Clients.ActiveDirectory.Platform\bin\$(Configuration)\net45\Microsoft.IdentityModel.Clients.ActiveDirectory.Platform.pdb">
       <TargetFolder>net45</TargetFolder>
     </SourceFile>
-    <SourceFile Include="..\src\Microsoft.IdentityModel.Clients.ActiveDirectory.Platform\bin\$(Configuration)\net45\Microsoft.IdentityModel.Clients.ActiveDirectory.Platform.xml"
+    <SourceFile Include="$(MSBuildThisFileDirectory)..\src\Microsoft.IdentityModel.Clients.ActiveDirectory.Platform\bin\$(Configuration)\net45\Microsoft.IdentityModel.Clients.ActiveDirectory.Platform.xml"
                 Condition="$(CopyXmlDocFiles)=='true'">
     <TargetFolder>net45</TargetFolder>
     </SourceFile>
 
-    <SourceFile Include="..\src\Microsoft.IdentityModel.Clients.ActiveDirectory.Platform\bin\$(Configuration)\Xamarin.iOS10\Microsoft.IdentityModel.Clients.ActiveDirectory.Platform.dll">
+    <SourceFile Include="$(MSBuildThisFileDirectory)..\src\Microsoft.IdentityModel.Clients.ActiveDirectory.Platform\bin\$(Configuration)\Xamarin.iOS10\Microsoft.IdentityModel.Clients.ActiveDirectory.Platform.dll">
       <TargetFolder>Xamarin.iOS10</TargetFolder>
     </SourceFile>
-    <SourceFile Include="..\src\Microsoft.IdentityModel.Clients.ActiveDirectory.Platform\bin\$(Configuration)\Xamarin.iOS10\Microsoft.IdentityModel.Clients.ActiveDirectory.Platform.pdb">
+    <SourceFile Include="$(MSBuildThisFileDirectory)..\src\Microsoft.IdentityModel.Clients.ActiveDirectory.Platform\bin\$(Configuration)\Xamarin.iOS10\Microsoft.IdentityModel.Clients.ActiveDirectory.Platform.pdb">
       <TargetFolder>Xamarin.iOS10</TargetFolder>
     </SourceFile>
-    <SourceFile Include="..\src\Microsoft.IdentityModel.Clients.ActiveDirectory.Platform\bin\$(Configuration)\Xamarin.iOS10\Microsoft.IdentityModel.Clients.ActiveDirectory.Platform.xml"
+    <SourceFile Include="$(MSBuildThisFileDirectory)..\src\Microsoft.IdentityModel.Clients.ActiveDirectory.Platform\bin\$(Configuration)\Xamarin.iOS10\Microsoft.IdentityModel.Clients.ActiveDirectory.Platform.xml"
                 Condition="$(CopyXmlDocFiles)=='true'">
       <TargetFolder>Xamarin.iOS10</TargetFolder>
     </SourceFile>
 
-    <SourceFile Include="..\src\Microsoft.IdentityModel.Clients.ActiveDirectory.Platform\bin\$(Configuration)\netcore45\Microsoft.IdentityModel.Clients.ActiveDirectory.Platform.dll">
+ <!--   
```

Comment:
Also commented out this configuration as netcore45 does not exist / is not generated in our targetplatforms.
We might want to change it by the one which is generated. would it be winrt?